py3status/modules/emerge_status.py

```diff
@@ -0,0 +1,139 @@
+# -*- coding: utf-8 -*-
+"""
+Emerge_status provides a short information on the current
+emerge status if there is an emerge process currently running
+
+Configuration parameters:
+    cache_timeout: how often we refresh this module in second (default 0)
+    format: Display format to user (default 'EMRG: {current} / {total}')
+    hide_if_zero: Don't show in bar if there is no emerge running (default False)
+
+Format placeholders:
+    {current} Number of package that is currently emerged
+    {total} Total number of packages that will be emerged
+    {category} Category of the currently emerged package
+    {pkg} Name of the currently emerged packaged
+    {action} Current emerge action
+
+Format examples:
+     {current} / {total}'
+    Result: "3 / 14"
```

Comment:
Wrap this part in `Examples: """ (docstrings) """`. See other modules for some examples. (ha)